recipes/mp++/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "0.2" %}
+
+package:
+  name: mp++
```

Comment:
It sounds like `+` is also something not explicitly allowed by `conda` (at least according to the docs).

> The name of a package, without any reference to a particular version. Conda package names are normalized, and may contain only lowercase alpha characters, numeric digits, underscores, hyphens, or dots. In usage documentation, these will be referred to by package_name.

ref: https://conda.io/docs/building/pkg-name-conv.html